recipes/axel/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+./configure --disable-Werror --prefix=$PREFIX
+make
+make install
```

Comment:
Is there a make check?

Reply:
The only reference to `make check` I could find was in the generic documentation, so I'm not sure if they've implemented it. Worth a try.

https://github.com/axel-download-accelerator/axel/blob/bd8b313d2e8b2dcf4d934eaa2985424f961112b9/INSTALL#L61